### PR TITLE
adding very rough draft of introduction.

### PR DESCRIPTION
**Pull request by jaybee84**
### Purpose
just adding a few thoughts for a skeleton to build the intro on

### Directions for reviewers
This is not ready for review yet, but wanted to start a draft to start thinking of possible things to add to the intro.

#### Is there anything that you want to discuss further?
Not yet

#### Is the pull request ready for review?

No

### Manuscript checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [ ] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#manubot-markdown)]
- [ ] All citations follow the [Manubot citation instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#citations)
- [ ] All changes or additions to tables follow the [Manubot table instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#tables)
- [ ] All figures follow the [Manubot figure instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#figures)
- [ ] All terms that need to be added to glossary have been added to #50 
- [ ] There are no new spelling errors identified by the Manubot spellchecker

### CODE REVIEW

File: content/02.intro.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
 ## Introduction {.page_break_before}
 
-This is an introductory section...
\ No newline at end of file
+Machine learning is gaining momentum in analysis of biomedical data as the data collection leans increasingly towards high dimensionality data and the algorithmic applications become increasingly transparent and interpretable. 
+
+Application of machine learning to any dataset is complicated but the complexity of applying these to biomedical dataset is enormous. 
+The proper application and interpretation requires depth of knowledge not only in the biomedical domain but also a clear understanding of the methods and the underlying assumptions of each of the methods.
+
+Rare disease has not benefitted significantly from machine learning applications yet due to various reasons including lack of statistical power in dataset size, heterogeneity in available data, and sensitivity of machine learning methods to misinterpretation in view of small datasets.
```
Comment (jaclyn-taroni) [inline content/02.intro.md]:
```suggestion
Rare disease research has not yet significantly benefited from machine learning applications for various reasons, including lack of statistical power in dataset size, heterogeneity in available data, and sensitivity of machine learning methods to misinterpretation in view of small datasets. 
```

File: content/02.intro.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+The proper application and interpretation requires depth of knowledge not only in the biomedical domain but also a clear understanding of the methods and the underlying assumptions of each of the methods.
+
+Rare disease has not benefitted significantly from machine learning applications yet due to various reasons including lack of statistical power in dataset size, heterogeneity in available data, and sensitivity of machine learning methods to misinterpretation in view of small datasets. 
+However recent advances in the methodologies to accomodate rarity of samples and increased transparency in model outputs have encouraged application of machine learning in rare disease.
+We anticipate higher occurrence of such applications in the near future and aim to highlight the current state-of-art in this perspective.
+
+Application of machine learning to any kind of data consists of the following major steps: (1) data evaluation and question formulation, (2) selection of normalization/dimension reduction to mitigate technical differences, (3) selection of appropriate algorithms to answer the formulated question, (4) evaluation of the answers generated by the algorithm.
```
Comment (jaclyn-taroni) [inline content/02.intro.md]: `dimension reduction to mitigate technical differences` Not sure about this part specifically. Dimension reduction could be for feature learning. Can you tell me more about what you meant?

File: content/02.intro.md [hunk 3]
```diff
@@ -1,3 +1,16 @@
 ## Introduction {.page_break_before}
 
-This is an introductory section...
\ No newline at end of file
+Machine learning is gaining momentum in analysis of biomedical data as the data collection leans increasingly towards high dimensionality data and the algorithmic applications become increasingly transparent and interpretable.
```
Comment (jaclyn-taroni) [inline content/02.intro.md]:
```suggestion
Machine learning is gaining momentum in biomedical data analysis as data collection is increasingly high-throughput and algorithms or approaches become more transparent and interpretable. 
```

File: content/02.intro.md [hunk 4]
```diff
@@ -1,3 +1,16 @@
 ## Introduction {.page_break_before}
 
-This is an introductory section...
\ No newline at end of file
+Machine learning is gaining momentum in analysis of biomedical data as the data collection leans increasingly towards high dimensionality data and the algorithmic applications become increasingly transparent and interpretable. 
+
+Application of machine learning to any dataset is complicated but the complexity of applying these to biomedical dataset is enormous. 
+The proper application and interpretation requires depth of knowledge not only in the biomedical domain but also a clear understanding of the methods and the underlying assumptions of each of the methods.
```
Comment (jaclyn-taroni) [inline content/02.intro.md]:
```suggestion
Application of machine learning to any dataset poses challenges, but the application to biomedical data and subsequent interpretation requires depth of knowledge not only in the biomedical domain but also a clear understanding of the methods and their underlying assumptions.
```

As you know, I am always worried about word count 😄

File: content/02.intro.md [hunk 5]
```diff
@@ -1,3 +1,16 @@
...
+
+Rare disease has not benefitted significantly from machine learning applications yet due to various reasons including lack of statistical power in dataset size, heterogeneity in available data, and sensitivity of machine learning methods to misinterpretation in view of small datasets. 
+However recent advances in the methodologies to accomodate rarity of samples and increased transparency in model outputs have encouraged application of machine learning in rare disease.
+We anticipate higher occurrence of such applications in the near future and aim to highlight the current state-of-art in this perspective.
```
Comment (jaclyn-taroni) [inline content/02.intro.md:8]: I would switch the order of these points.

File: content/02.intro.md [hunk 6]
```diff
@@ -1,3 +1,16 @@
...
+
+Application of machine learning to any kind of data consists of the following major steps: (1) data evaluation and question formulation, (2) selection of normalization/dimension reduction to mitigate technical differences, (3) selection of appropriate algorithms to answer the formulated question, (4) evaluation of the answers generated by the algorithm. 
+Each of these steps require the user to choose from a variety of methodologies to apply.
```
Comment (jaclyn-taroni) [inline content/02.intro.md]:
```suggestion
Each of these steps require the practitioner to choose from a variety of methodologies to apply. 
```
? User makes me think of software but that may only be me!

**Review by jaclyn-taroni — APPROVED** [review]
Had a few comments - but sounds like we want to start getting things merged so we can go back through so I will approve ✅

File: content/02.intro.md (hunk 2 above)
Comment (jaybee84) [reply content/02.intro.md]: Fair point. In this instance, I was referring mostly to exploratory dimension reduction analyses like PCA, tSNE etc which are used to explore existing groups in the data (to test if they are technical or disease-related) and are used in conjunction with normalization techniques. 

But important to state the feature learning side of dimension reduction as well. In this case, it was implied in the selection of algorithm point (where various dimension reduction steps may be used for clustering/classifying data). I can add that more explicitly.

**Pull request merged**